benchmarks/java/src/main/java/com/google/protobuf/ProtoCaliperBenchmark.java

```diff
@@ -22,6 +23,7 @@
 import java.util.ArrayList;
 import java.util.List;
 
+@VmOptions("-XX:-TieredCompilation")
```

Comment:
Why do we need this?

Reply:
It seems a bug for caliper that it set CICompilerCount to 1, for TieredCompilation this value must be at least 2. So I just disable TieredCompilation.

Reply:
So it fails on kokoro but not locally? Can we set some environment variable to make it also work on kokoro? I'm worried that disabling TieredCompilation would change the performance characteristics and make the benchmark result less meaningful.

Reply:
I think it fail on kokoro is the version of java is different from local one. In fact caliper's fix after 1.0-beta-2 for that is just [disabling TieredCompilation](https://github.com/google/caliper/blob/97a906bd6d91e9e0a30d413966b3a11469cb9ebf/caliper-runner/src/main/java/com/google/caliper/runner/target/Jvm.java#L39).  I think it's OK to do that

Reply:
Please add comments for why this is needed and link the caliper source as well.

Reply:
Done